### PR TITLE
Add CI to microkelvin

### PR DESCRIPTION
**Pull request by ZER0**
- Add `wrap_comments` to rustfmt
 - Add rust-toolchain for consistent behavior with CI
    
Resolves: #2

### CODE REVIEW

File: src/branch.rs [hunk 1]
```diff
@@ -45,6 +45,10 @@ where
         self.0.len()
     }
 
+    pub fn is_empty(&self) -> bool {
+        self.0.is_empty()
+    }
+
```
Comment (krl) [inline src/branch.rs:51]: Why is this needed? It will always return false.
Please do a separate PR for code changes.

**Review by krl — CHANGES_REQUESTED** [review]

File: src/branch.rs (hunk 1 above)
Comment (ZER0) [reply src/branch.rs:51]: It's needed because you added `len` method, and it's good practice have always both.

> It is good custom to have both methods, because for some data structures, asking about the length will be a 
> costly operation, whereas .is_empty() can usually answer in constant time. Also it used to lead to false positives
> on the len_zero lint – currently that lint will ignore such entities.

If it always return false we can just make it false.

It's part of this PR since we're checking the warnings and the codestyle before commit code, and it will be likely part of the CI.
If you want to have a separate PR, it means I have to address those nits before the CI, it's fine by me.

Comment (krl) [reply src/branch.rs:51]: Yeah i would make a separate PR since it adds API that is not in the released version.

Comment (ZER0) [reply src/branch.rs:51]: Filed https://github.com/dusk-network/microkelvin/pull/6

**Review by krl — APPROVED** [review]
LGTM
